js/letters.js

```diff
@@ -17,8 +17,24 @@ var counter = {
 
 function countLetters(counter, sample_text){
   // FIX ME
+  if (sample_text.length === 0){
+    return counter;
+  }
+  for (let property in counter){
```

Comment:
Insert newline above the for loop

Reply:
You don't need to use a for loop for this. There is a way to check if the character you're looking at is within the object without having to check every key in the object.